src/features/content-explorer/content-explorer/ContentExplorerActionButtons.js

```diff
@@ -88,7 +88,7 @@ const ContentExplorerActionButtons = ({
     };
 
     return (
-        <div className="content-explorer-action-buttons-container" {...actionButtonsProps}>
+        <div className="modal-actions" {...actionButtonsProps}>
```

Comment:
I'm not sure if these classes are swappable.

There are nested styles in `content-explorer-action-buttons-container` that are not in `modal-actions` such as `status-message` and `content-explorer-copy-button`:
https://github.com/box/box-ui-elements/blob/master/src/features/content-explorer/content-explorer/ContentExplorer.scss#L83-L103

The `position: relative;` is also important here so we'd need to confirm that none of the child components that are positioned `absolute` are effected.

Let's also verify the impact in the parent apps that might have repo specific styles for this class.

Reply:
Did some research on switching these two. Seems the `.status-message` is used by the MultiSelect variant of the ContentExplorer. It displays the the number of items selected.

Screen Shot 2022-07-08 at 9.25.40 AM

The other class `.content-explorer-copy-button` sets the `margin-right: 0`. Which is setting the margin on the right most item. The `.modal-actions:` already applies this as:

.modal-actions .btn:last-of-type {
    margin-right: 0;
}
from (https://github.com/box/box-ui-elements/blob/master/src/components/modal/Modal.scss#L117)

I'd consider switching to using `.modal-actions` as a "responsiveness change" since we want the buttons to be on the bottom on small screens. For that reason I've made it conditional to the `isResponsive` prop so it's use will be strictly opt-in.